cookbooks/mediawiki/templates/default/LocalSettings.php.erb

```diff
@@ -98,6 +98,23 @@ $wgFileExtensions[] = 'osm';
 	$wgFileExtensions[] = '<%= mw_extra_file_extension %>';
 <% end -%>
 
+# Add OSM XML file format per http://www.iana.org/assignments/media-types/media-types.xhtml
+# Shout out to Paul Norman for reserving this.
+$wgXMLMimeTypes[] = 'osm' => 'application/vnd.openstreetmap.data+xml';
```

Comment:
Sorry but I don't know PHP at all, so can you explain what this (which looks a little odd) actually does? Am I right in assuming that this adds a new entry to an existing hash?

Reply:
This line is actually incorrect - you are right for bringing this up.

I meant to just submit the configuration below instead. Although you mention keeping this dynamic, so I will re-visit this. (and correct it)